### DISCUSSION TITLE
Better visibility for unread and important unread articles

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/FreshRSS#5910] [Show and tell]
Hi,

I want to show you how I tweaked a bit the UI to have a better overview of unread (1) and important unread (2) feeds:

![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/a0de5e5d-d4ca-4f8e-9d05-e6d5ca411059)

While I use FreshRSS on my mobile device often I disabled the (un)read button to save the space. The dot helps me to see the unread articles. The exclamation mark highlights the important feed articles.

What do you think: Is it worth to implement it into FreshRSS or is it just a nice to have thing that could be used via CustomCSS extension?

That is the CSS code:
```
.flux.not_read .flux_header .item.title .item-element::before {
	content: '• ';
}

.flux.not_read[data-priority="20"] .flux_header .item.title .item-element::before {
content: '! ';
}
```

Have not checked yet if it works in other themes. I use it with the Nord theme.

**Comment by Frenzie** [comment]
In most other themes read/unread is also differentiated by color, not just by that icon.

**Reply by Frenzie** [reply]
But personally I prefer the current default and I wouldn't want to double it fwiw.
